be/src/agent/task_worker_pool.cpp

```diff
@@ -1024,6 +1025,18 @@ void* TaskWorkerPool::_update_tablet_meta_worker_thread_callback(void* arg_this)
                 case TTabletMetaType::INMEMORY:
                     // This property is no longer supported.
                     break;
+                case TTabletMetaType::ENABLEPERSISTENTINDEX:
+                    VLOG(1) << "update tablet:" << tablet->tablet_id()
+                            << " enable_persistent_index:" << tablet_meta_info.enable_persistent_index;
+                    tablet->set_enable_persistent_index(tablet_meta_info.enable_persistent_index);
```

Comment:
Is this persisted to metapb?

Reply:
Also maybe change VLOG to LOG

Reply:
> Is this persisted to metapb?

Yes. It will call `tablet->save_meta()` to persistent tablet_meta after modify this in memory